### DISCUSSION TITLE
how to use 74AHCT125 to make ws2812b work?

### DISCUSSION THREAD

**Discussion by eduardofegatti** [awawa-dev/HyperHDR#177] [Q&A]
First congratulations for the amazing work! I'm dreaming of making it work I'm from Brazil
The image from the capture card is recognized by the hyperHDR software but the tape does not respond to the images... Either it stays on all the time or it only lights up 5 LEDs

I'm at work soon, I'll send the logs

I have:

1 x ws2812b tape (on my tv there were 73 top and down leds, 42 left and 42 right leds, total 228 leds

1x Raspberry PI 3
Version 17 hyperHDR

1 x Source 5v 20A 100W https://pt.aliexpress.com/item/32679309139.html?spm=a2g0s.9042311.0.0.2742b90ajnzRVn

1 video capture https://pt.aliexpress.com/item/1005001592294499.html?spm=a2g0s.9042311.0.0.2742b90a9aziQP
navceker 3x1
I made the default setting, my ribbon lights up colored when raspberry restarts and erases and then stays lit all white.

I bought the level shifter but I don't have the slightest idea where the wires fit on the level shifter pins, I looked here if someone posted a picture, but I couldn't find it.

I found this way would it be this way?

https://www.thegeekpub.com/15990/wiring-ws2812b-addressable-leds-to-the-raspbery-pi/

Sorry if I broke any rules by sending links . Thanks

**Comment by awawa-dev** [comment]
Hi
You don't have any reaction from LED strip when you try to play an effect or set color in the remote tab? And no erros in the logs (they are in red)?

That wiring from link you posted for SN74AHCT125N is OK. Mine is the same but I use ESP device instead of Raspberry Pi and that's the recommended way but your setup should also work with level shifter.
More pictures:
![obraz](https://user-images.githubusercontent.com/69086569/146442829-b3b81958-e2af-4f85-8002-7458bc9563e5.png)
![SN74AHCT125N_level_shifter](https://user-images.githubusercontent.com/69086569/146442851-47ff5060-8da1-4f9f-9ce4-cec852a6d775.png)
![level](https://user-images.githubusercontent.com/69086569/169654166-cb73d257-633f-403c-8205-3f0532653003.jpg)

Regards
Awawa
